src/index.js

```diff
@@ -0,0 +1,150 @@
+module.exports = {
+  env: {
+    browser: true,
+    node: true,
+    "shared-node-browser": true,
+    "cypress/globals": true,
+    "jest/globals": true,
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
+    "plugin:react-hooks/recommended",
+    "plugin:cypress/recommended",
+    "plugin:jest/recommended",
+    "plugin:testing-library/react",
+    "plugin:json/recommended",
+    "plugin:prettier/recommended",
+    "prettier/react",
+    "prettier/@typescript-eslint",
+  ],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    project: "tsconfig.json",
+    sourceType: "module",
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+  plugins: [
+    "@typescript-eslint",
+    "@typescript-eslint/tslint",
+    "react",
+    "cypress",
+    "jest",
+    "testing-library",
+    "prettier",
+    "import",
+    "json",
+  ],
+  settings: {
+    react: {
+      version: "detect",
+    },
+  },
+  overrides: [
+    {
+      files: ["*.js", "*.jsx"],
+      rules: {
+        "@typescript-eslint/no-var-requires": "off",
+        "@typescript-eslint/explicit-function-return-type": "off",
+      },
+    },
+    {
+      files: ["**/cypress/**/*"],
+      rules: {
+        "jest/expect-expect": "off",
+        "jest/valid-expect-in-promise": "off",
+        "jest/valid-expect": "off",
+        "jest/no-standalone-expect": "off",
+        "testing-library/await-async-query": "off",
+      },
+    },
+    {
+      files: [
+        "**/__tests__/next/**/*.test.tsx",
+        "**/__tests__/pages/**/*.test.tsx",
+        "**/__tests__/pages/**/*.test.jsx",
+        "**/components/**/*.test.tsx",
+        "**/components/**/*.test.jsx",
+      ],
+      rules: {
+        // off because of testing library
```

Comment:
I can not follow the comment.

Reply:
Those two linter rules ensure that:
- your test always has an `expect`
- an expect in your test always have a condition

With the testing library, this is no longer the case since testing for the existence of an element can simply use matcher helpers without an expect at all.